src/app/demo/typescript/typescript.component.ts

```diff
@@ -10,6 +10,36 @@ import {ng2tsConfig} from '../../../../ng2ts/ng2ts';
 })
 
 export class TypescriptComponent {
+
+  introJsBefore() {
+    let squiggly;
+
+    squiggly = document.querySelector('.highlighted-code');
```

Comment:
Yes, it _should_... third time's the charm!

oh really? 🤔🤔🤔

ah, well webstorm is too expensive :);;;;;;;;;;;;;